PlayCover/ViewModel/NetworkVM.swift

```diff
@@ -5,37 +5,6 @@
 //  Created by Isaac Marovitz on 09/10/2022.
 //
 
-// import Network
```

Comment:
This should be done in another PR.

Reply:
This code never seems to work and contains the usage of DispatchQueue, hence my reason for deleting it.

Reply:
It is unrelated to the main changes of this PR.

Reply:
Either that or commit squashing.